packages/fiori/src/types/UploadCollectionDnDMode.js

```diff
@@ -0,0 +1,29 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
```

Comment:
The enums should be documented as well. Use the AvatarShape.js for reference and it is ok to be public

Reply:
I added jsdoc to this type, but it is only used in a private property "_dndOverlayMode" and thats why I kept it private